.codecov.yml

```diff
@@ -2,5 +2,5 @@ coverage:
   status:
     patch:
       default:
-        threshold: 0.0%
+        threshold: 2.0%
```

Comment:
> A build will fail if the code coverage reduction is greater than 2%.

Isn't that actually saying that the patch shouldn't have less than 98% line coverage? Meaning the above statement would only be true if the patch changes every single line. A patch that has 98% coverage does only reduce the project coverage by << 1% i.e.  a patch with 98% coverage that touches 0.5% of the lines reduces the project line coverage by (0.5 * 0.02)% which means the project coverage would only be reduced by 0.01. Meaning any patch that touches more or equal to 0.5% of the lines and passes coverage/patch would still fail coverage/project.

Reply:
You are right, we need to make a distinction between [the patch](https://docs.codecov.io/docs/commit-status#section-patch-status) and [the project](https://docs.codecov.io/docs/commit-status#section-project-status).
I have looked at https://github.com/WordPress/gutenberg/pull/11198/files. What do you think of:
```jsx
coverage:
  status:
    project:
      default:
        target: auto
        threshold: 5%
    patch: off
comment: false
```
?

Reply:
Wordpress has iteration multiple time on a correct configuration. I would trust them with that. Reading the issue, the motivation for removing codecov was an issue with collecting the coverage data accurately. It's not yet an issue we have at our scale. I'm proposing 5%, they were using 0.5%, looking at #13676 and #13664, 0.5% is good enough, maybe 1%?

Reply:
Or maybe we could have another threshold for a patch, like 25%.

Reply:
I'd be super happy with that. Previous config would punish small patches. We should make it a habit to review the coverage report manually though.